### ISSUE TITLE
Some scenes run twice regardless of the command pressed

### ISSUE THREAD

**Issue by gauthrs** [telegraf/telegraf#1656]
Hello!
When you press /start everything works fine. But if I click on any button in the defaultKeyboard(), then I will get the start message again. At the same time, if I again click on the button in the defaultKeyboard(), then everything works out correctly.
The same situation is with the Back button.
Where is my error?

```
const getBackKeyboard = () => {
    const backKeyboard = Markup.keyboard([buttons.back]);
    return backKeyboard.resize();
}

const getDafaultKeyboar = () => {
    const defaultKeyboard = Markup.keyboard([
        [buttons.how_buy_toncoin, buttons.wallet]
    ]);
    return defaultKeyboard.resize();
}

const startScene = new WizardScene(
    'START_SCENE',
    async (ctx) => {
        await ctx.replyWithHTML(text.welcome, getDafaultKeyboar());
        return ctx.scene.leave(ctx);
    }
);

const walletScene = new WizardScene(
    'WALLET_SCENE',
    async (ctx) => {
        await ctx.reply(`text.wallet_how_balance`, getBackKeyboard());
        return ctx.wizard.next();
    },
    async (ctx) => {
        const wallet = ctx.message.text;

        await fetchAddressInformation(wallet).then(({ result }) => {
                const balance = (result.balance * 1e-9).toFixed(2);

                ctx.reply(`text.balance`, {parse_mode: 'HTML'});
                return ctx.scene.leave(ctx);
        }).catch(() => {
            ctx.reply(`text.wallet_incorrect`);
            return;
        });
    }
);
walletScene.hears(buttons.back, leave());

const howBuyToncoinScene = new WizardScene(
    'HOW_BUY_TONCOIN_SCENE',
    async (ctx) => {
        await ctx.reply(text.how_buy_toncoin, {
            parse_mode: 'HTML',
            disable_web_page_preview: true
        });
        return ctx.scene.leave();
    }
);

const stage = new Stage([ walletScene, startScene, howBuyToncoinScene ]);

const bot = new Telegraf(process.env.TELEGRAM_BOT_TOKEN);
bot.use(session(), stage.middleware());

bot.start(async (ctx) => ctx.scene.enter('START_SCENE'));
bot.hears(buttons.wallet, async (ctx) => ctx.scene.enter('WALLET_SCENE'));
bot.hears(buttons.back, (ctx) => ctx.reply(text.welcome_back, getDafaultKeyboar()));
bot.hears(buttons.how_buy_toncoin, async (ctx) => ctx.scene.enter('HOW_BUY_TONCOIN_SCENE'));
bot.launch();
```

**Comment by gauthrs** [comment]
I forgot to write that this happens only during the initial launch of the bot. But with the Back button, this always happens.

**Comment by iamguid** [comment]
Have the same issue, it is critical

**Comment by MKRhere** [comment]
This may have been due to #1372, fixed in #1713. Closing for housekeeping. Do reopen if this happens after v4.12.
